scm/issue.go

```diff
@@ -43,11 +43,13 @@ type (
 
 	// Comment represents a comment.
 	Comment struct {
-		ID      int
-		Body    string
-		Author  User
-		Created time.Time
-		Updated time.Time
+		ID       int
+		Body     string
+		Author   User
+		Created  time.Time
+		Updated  time.Time
+		Type     string
+		Metadata interface{}
```

Comment:
I recommend we remove `type` and `metadata` at this time until we have a design for all providers (github, gitlab, etc).

Reply:
done